### DISCUSSION TITLE
environment variable HELM_MAX_HISTORY of helm is not considered by helmfile,it is taking default value of history max

### DISCUSSION THREAD

**Discussion by abrarcv170** [helmfile/helmfile#305] [Questions]
environment variable HELM_MAX_HISTORY of the helm is not considered by helmfile, it is taking the default value of --history-max (10).

**Comment by yxxhero** [comment]
@abrarcv170 PR is welcome.

**Comment by yxxhero** [comment]
@abrarcv170 https://helmfile.readthedocs.io/en/latest/#configuration  `historyMax`

**Reply by abrarcv170** [reply]
@yxxhero why  it  is not considering the HELM_MAX_HISTORY  environment variable ?, I tested old versions of helmfile (v0.90.8).HELM_MAX_HISTORY helm env is working fine.

**Reply by yxxhero** [reply]
I don't know much about the previous logic. I hope it can help you by configuration.

**Reply by abrarcv170** [reply]
I am managing more than 200 releases ,so changing configuration in each release is not ideal ,

**Reply by yxxhero** [reply]
how many helmfiles do you mange? @abrarcv170

**Reply by yxxhero** [reply]
can you use `bases` config key, and include it for all you release?

**Reply by abrarcv170** [reply]
I am managing more than  200 helmfiles

**Comment by yxxhero** [comment]
@abrarcv170 PR is welcome.
